### PR TITLE
Simple colorscheme support

### PR DESCRIPTION
**Pull request by bloodearnest**
Disables use of italics/bold/dim text formatting in colorisation, and better support for light terminal backgrounds

### CODE REVIEW

**Comment by bloodearnest** [comment]
Hey @nottrobin, can I get you to review this branch, and test it fixes your color issues? I could not reproduce locally at all, so I added a TALISKER_COLOR=simple env var which uses simpler colorscheme, which hopefully addresses your issue.

Should be able to just do:

pip install git+https://github.com/canonical-ols/talisker@simple-colorscheme

**Comment by bloodearnest** [comment]
Fixes #152

**Comment by nottrobin** [comment]
Sorry this took me so long to test @bloodearnest. Yes that looks much better, when testing with `TALISKER_COLOR=simple` (the default still looks just as bad):

image

That's a good solution, I'm happy to define `TALISKER_COLOR=simple` locally.

File: docs/development.rst [hunk 1]
```diff
@@ -10,23 +10,22 @@ Talisker's default configuration is designed for production usage, e.g.:
...
-For Gunicorn, devel mode means a few more things:
+For Gunicorn, DEVEL mode means a few more things:
 
  - enables access logs to stderr
  - sets timeout to 99999, to avoid timeouts when debugging
  - it enables auto reloading on code changes
 
-Also, for convenience, it you manually set Gunicorn's debug level to DEBUG, when
-in devel mode, Talisker will actually log debug level messages to stderr.
+Also, for developer convenience, it you manually set Gunicorn's debug level to DEBUG, when
```
Comment (nottrobin) [inline docs/development.rst]: I think this is a typo - should be "if you manually..."

**Comment by nottrobin** [comment]
For the record, default colour scheme still looks like this:

image

**Pull request merged**